spec/models/lettings_log_spec.rb

```diff
@@ -112,10 +112,17 @@
     end
 
     it "validates tenancy type" do
-      expect(validator).to receive(:validate_fixed_term_tenancy)
       expect(validator).to receive(:validate_other_tenancy_type)
     end
 
+    it "validates tenancy length" do
```

Comment:
I'm not really sure this test is useful - but I think that's part of taking a wider look at our tests so I've updated it to match the current setup for the moment.